staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -53,6 +55,8 @@ const (
 	// 34 chose as a number close to 30 that is likely to be unique enough to jump out at me the next time I see a timeout.
 	// Everyone chooses 30.
 	requestTimeout = 34 * time.Second
+	// TODO(roycaihw): stop serving requests with duplicate owner reference entries in 1.24
+	DuplicateOwnerReferencesWarning = ".metadata.ownerReferences contains duplicate entries; API server dedups owner references in 1.20+, rejects such requests in 1.24+; please fix your requests"
```

Comment:
the dedup behavior is deprecated. Rejecting requests in 1.24 gives us one year to remove this behavior

https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior